operator.yaml

```diff
@@ -54,13 +54,13 @@ spec:
             - name: SERVICE_ACCOUNT_NAME
               value: hazelcast
             - name: RELATED_IMAGE_HAZELCAST
-              value: hazelcast/hazelcast-enterprise:3.12.5
+              value: hazelcast/hazelcast-enterprise:4.0
             - name: RELATED_IMAGE_MANCENTER
-              value: hazelcast/management-center:3.12.6
+              value: hazelcast/management-center:4.0
           resources:
             limits:
-              cpu: "1"
+              cpu: "0.1"
               memory: 512Mi
             requests:
-              cpu: "0.5"
+              cpu: "0.1"
```

Comment:
How did you come up with those figures?

Reply:
We didn't do any measurements, I took them from some Kubernetes example. To be honest, they will depend on your environment, but `0.5` was definitely too high, you don't need that much to run operator.